pkg/apitype/policy.go

```diff
@@ -107,3 +130,12 @@ type GetStackPolicyPacksResponse struct {
 	// RequiredPolicies is a list of required Policy Packs to run during the update.
 	RequiredPolicies []RequiredPolicy `json:"requiredPolicies,omitempty"`
 }
+
+// ConfigurePolicyPackRequest defines the configuration for a Policy
+// Pack.
+type ConfigurePolicyPackRequest struct {
```

Comment:
Let's remove this type from the PR.

For the initial version, we'll just support configuring the policy pack from within the Service UI. We will support CLI-gestures later, and we can add the types from `service_apitype` to this repo at that time. (Which would make it easier to iterate for the time being.)

When we do go to commit the `ConfigurePolicyPackRequest` to a repo, we'll need to use a "patch config" type approach. Because as-is, this "all or nothing", "overwrite all configuration on each request", has lots of problems related to data races and is very cumbersome in practice.

Instead, we'd probably want to have separate requests for just updating the configuration _for a single policy pack_, or even adding/updating/removing specific configuration values. Something like:

```
// ConfigurationPolicyPackRequest replaces the stored configuration data for a single
// policy rule.
type ConfigurationPolicyPackRequest struct {
    PolicyRule string
    Configuration PolicyConfiguration
}
```